scripts/serve/serve-static.js

```diff
@@ -9,7 +9,7 @@ const displayServer = (localAddress) => {
   Logger.log(`* Local:            ${Logger.emphasis(localAddress)}`);
 };
 
-const dirExists = site => (fs.existsSync(site) && fs.lstatSync(site).isDirectory());
+const dirExistsWithContent = site => glob.sync(`${site}/**/*`).length > 0;
```

Comment:
I think this would be cheaper just to read the directory with fs.readdirsync, the only case it wouldn't handle is if that directory contains empty directories. Is that a case we're worried about?

https://nodejs.org/api/fs.html#fs_fs_readdirsync_path_options

Reply:
I hadn't thought about empty nested dir. I don't think its a much of a concern though since the clean plugin will blow away the build dir. I honestly just didn't spend too much time looking into the issue. Will look into the options/link you suggested

Reply:
Yes, I think reading the dir content and checking the length is a better option

Reply:
updated: https://github.com/cerner/terra-toolkit/pull/298/commits/9ff9152b60a139e62cc5c4e31afaddf077668ccf#diff-2d0e39e2b5431dc8d9df655f5fc376fbR12